langchain/src/vectorstores/chroma.ts

```diff
@@ -38,6 +38,18 @@ export class Chroma extends VectorStore {
     );
   }
 
+  async fromExistingCollection() {
```

Comment:
This method name is confusing, given you're creating a static method of the same name. Can you rename to eg. `_ensureCollection` ?

Reply:
Renamed to `ensureCollection` as agreed over discord.